pypeit/wavecalib.py

```diff
@@ -628,18 +635,35 @@ def build_wv_calib(self, arccen, method, skip_QA=False,
         elif method == 'reidentify':
             # Now preferred
             # Slit positions
-            arcfitter = autoid.ArchiveReid(arccen, self.lamps, self.par,
-                                           ech_fixed_format=self.spectrograph.ech_fixed_format, ok_mask=ok_mask_idx,
-                                           measured_fwhms=measured_fwhms,
-                                           orders=self.orders,
-                                           nonlinear_counts=self.nonlinear_counts)
+            arcfitter = autoid.ArchiveReid(
+                arccen, self.lamps, self.par,
+                ech_fixed_format=self.spectrograph.ech_fixed_format, 
+                ok_mask=ok_mask_idx,
+                measured_fwhms=self.measured_fwhms,
+                orders=self.orders,
+                nonlinear_counts=self.nonlinear_counts)
             patt_dict, final_fit = arcfitter.get_results()
+
+            # Save arxiv for redo later?
+            if self.par['echelle']: 
```

Comment:
Maybe it makes more sense to just add a method to ArchiveReid that generates these quantities, i.e. 

self.wave_soln_arxiv, self.arcspec_arxiv, self. arccen = arcfitter.get_arxiv()

Also, not sure why generating this is restricted to echelle. We rarely use ARchiveReid for multisit, although we could.

Reply:
good suggestion;  implemented